pkg/domain/infra/abi/parse/parse.go

```diff
@@ -78,6 +78,16 @@ func VolumeOptions(opts map[string]string) ([]libpod.VolumeCreateOption, error)
 					libpodOptions = append(libpodOptions, libpod.WithVolumeDisableQuota())
 					// set option "NOQUOTA": "true"
 					volumeOptions["NOQUOTA"] = "true"
+				case "timeout":
+					if len(splitO) != 2 {
+						return nil, errors.Wrapf(define.ErrInvalidArg, "gid option must provide a GID")
```

Comment:
```suggestion
						return nil, errors.Wrapf(define.ErrInvalidArg, "gid option must provide a GID")
```
Is this error message correct, if the case is testing for timeout? Looks like a cut and paste error.